assets/src/dashboard/app/views/templateDetails/index.js

```diff
@@ -20,10 +20,10 @@
 import { useCallback, useEffect, useMemo, useState } from 'react';
 import { trackEvent } from '@web-stories-wp/tracking';
 import { useSnackbar } from '@web-stories-wp/design-system';
+import { clamp } from '@web-stories-wp/animation';
```

Comment:
I think this `clamp` util should not be imported from the animation package.

Let's instead _copy_ it to `dashboard/utils`

Reply:
@BrittanyIRL Any thoughts on this?